roles/tests-integration/tests/translator_integration.rs

```diff
@@ -44,3 +44,18 @@ async fn translation_proxy() {
         )
         .await;
 }
+
+#[tokio::test]
+async fn jds_should_not_panic_if_jdc_shutsdown() {
+    let (_tp, tp_addr) = start_template_provider(None).await;
+    let (_pool, pool_addr) = start_pool(Some(tp_addr)).await;
+    let (_jds, jds_addr) = start_jds(tp_addr).await;
+    let (jdc, jdc_addr) = start_jdc(pool_addr, tp_addr, jds_addr).await;
+    jdc.shutdown();
+    tokio::time::sleep(std::time::Duration::from_secs(10)).await;
+    let ret = tokio::net::TcpListener::bind(jdc_addr).await;
+    assert!(ret.is_ok());
```

Comment:
this assert can be moved to the line above it 
`assert!(tokio::net::TcpListener::bind(jdc_addr).await.is_ok());`